### PR TITLE
initial release

### PR DESCRIPTION
**Pull request by aleXOR2**
@mikitachab please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,113 @@
...
+[pytest]
+addopts = --pylint
+```
+
+Acknowledgements
+================
+
+This code is a fork of 
+[pytest-pylint](https://github.com/carsongee/pytest-pylint.git) 0.14.0
+>>>>>>> initial release
```
Comment (mikitachab) [inline README.md]: git merge rubbish????

**Review by mikitachab — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,37 +1,125 @@
...
+
+Prerequisites
+=============
+
+* Linux OS
+* SVN 1.8+ (for svn repo linting)
+* Terminal locale set as UTF-8
+
+   Plugin uses terminal locale and assumes that it is set as UTF-8 encoding (en_GB.utf8, pl_pl.utf8, ru_RU.utf8, etc)
+* Python3.5+
```
Comment (mikitachab) [inline README.md]: Python3.5+ -> Python 3.5+

File: test/test_pytest_pylint_xdist_vcs.py [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+
+@pytest.fixture
+def file_with_multiple_tests(testdir):
+    """Fixture to create a python file for linting test"""
+    testdir.makepyfile(**{
+        'test_ok.py': """'''A simple test'''
+from __future__ import print_function
+
+
+TRUE = True
```
Comment (mikitachab) [inline test/test_pytest_pylint_xdist_vcs.py:121]: ?

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+exclude: ^(test/mocked_repo/)
+repos:
+  - repo: https://github.com/PyCQA/pylint
+    rev:  d0a597b
+    hooks:
+      - id: pylint
+        language_version: python3.6
```
Comment (mikitachab) [inline .pre-commit-config.yaml:7]: README says Python 3.5+

File: test/test_pytest_pylint_xdist_vcs.py (hunk 3 above)
Comment (aleXOR2) [reply test/test_pytest_pylint_xdist_vcs.py:121]: Auto generated test case. I aim to create assets folder during the refactoring upon the 0.1.0 release

File: README.md (hunk 2 above)
Comment (aleXOR2) [reply README.md]: Done

File: .pre-commit-config.yaml (hunk 4 above)
Comment (aleXOR2) [reply .pre-commit-config.yaml:7]: It is definite python version to run linter checks on CI. If you look at CI lint job - python 3.6 specified there

**Pull request merged**